### PR TITLE
ticket#7 patch /api/articles/article

### PR DESCRIPTION
**Pull request by Brynster99**

### CODE REVIEW

File: __tests__/endpoints.test.js [hunk 1]
```diff
@@ -90,3 +119,47 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('STATUS: 200, Returns updated object in new state', () => {
```
Comment (DominicFH) [inline __tests__/endpoints.test.js:115]: This test is looking good! I'd throw in another test just to ensure we can pass in a negative number to decrement the vote count

File: __tests__/endpoints.test.js [hunk 2]
```diff
@@ -90,3 +119,47 @@ describe('GET /api/articles/:article_id', () => {
...
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 150,
+          })
+        );
+      });
+  });
+
+  test('STATUS: 404, Returns not found if ID is invalid', () => {
```
Comment (DominicFH) [inline __tests__/endpoints.test.js]: I'd reword these sad path tests to be specific about what we're testing for as these two tests read as returning different messages if the id is invalid. Could maybe use wording like 'VALID but NON-EXISTENT article_id' and 'INVALID article_id'

**Review by DominicFH — CHANGES_REQUESTED** [review]
Some comments in the code about some adjustments to make but I'd also do some sad path tests about the request object itself. 

What if the value of inc_votes isn't a number? Also what if the request object doesn't have an inc_votes property?

File: __tests__/endpoints.test.js (hunk 1 above)
Comment (Brynster99) [reply __tests__/endpoints.test.js:115]: Have added negative inc_votes test - passing

File: __tests__/endpoints.test.js (hunk 2 above)
Comment (Brynster99) [reply __tests__/endpoints.test.js]: Have updated test names to reflect tests correctly

**Review by maddierosecodes — APPROVED** [review]
All comments addressed :)
